app/src/editor/shared/components/Subscription.jsx

```diff
@@ -53,7 +49,7 @@ const ClientProvider = withAuthKey(class ClientProvider extends Component {
 
     teardown() {
         const { client } = this.state
-        if (client) {
+        if (client && client.connection) {
```

Comment:
This looks like something the `streamr-client` should be checking?

Reply:
Yeah agreed